Tests/Common/Data/TickConsolidatorTests.cs

```diff
@@ -248,5 +248,49 @@ public void AggregatesNewTicksInPeriodWithRoundedTime()
             Assert.AreEqual(consolidated.Close, tick5.Value);
         }
 
+        [Test]
+        public void ProcessesTradeTicksOnly()
+        {
+            TradeBar consolidated = null;
+            var consolidator = new TickConsolidator(TimeSpan.FromMinutes(1));
+            consolidator.DataConsolidated += (sender, bar) =>
+            {
+                consolidated = bar;
+            };
+
+            var reference = new DateTime(2015, 06, 02);
+            var tick1 = new Tick
+            {
+                Symbol = Symbols.SPY,
+                Time = reference.AddSeconds(3),
+                Value = 200m
+            };
+            consolidator.Update(tick1);
+            Assert.IsNull(consolidated);
+
+            var tick2 = new Tick
+            {
+                Symbol = Symbols.SPY,
+                Time = reference.AddSeconds(10),
+                Value = 20000m,
+                TickType = TickType.OpenInterest
```

Comment:
Perhaps we should add a quote tick for good measure?

Reply:
Quote tick added 👍